grid-client/workloads/k8s.go

```diff
@@ -168,59 +186,13 @@ func (k *K8sCluster) ValidateToken() error {
 
 // ValidateIPranges validates NodesIPRange of master && workers of k8s cluster
 func (k *K8sCluster) ValidateIPranges() error {
-	if _, ok := k.NodesIPRange[k.Master.Node]; !ok {
-		return errors.Errorf("the master node %d does not exist in the network's ip ranges", k.Master.Node)
+	if _, ok := k.NodesIPRange[k.Master.NodeID]; !ok {
+		return errors.Errorf("the master node %d does not exist in the network's ip ranges", k.Master.NodeID)
 	}
-	for _, w := range k.Workers {
-		if _, ok := k.NodesIPRange[w.Node]; !ok {
-			return errors.Errorf("the node with id %d in worker %s does not exist in the network's ip ranges", w.Node, w.Name)
-		}
-	}
-	return nil
-}
-
-// ValidateNames validate names for master and workers
-func (k *K8sCluster) ValidateNames() error {
-	names := make(map[string]bool)
-	names[k.Master.Name] = true
 
 	for _, w := range k.Workers {
-		if _, ok := names[w.Name]; ok {
-			return errors.Errorf("k8s workers and master must have unique names: %s occurred more than once", w.Name)
-		}
-		names[w.Name] = true
-	}
-	return nil
-}
-
-// ValidateChecksums validate check sums for k8s flist
-func (k *K8sCluster) ValidateChecksums() error {
-	nodes := append(k.Workers, *k.Master)
-	for _, vm := range nodes {
-		if vm.FlistChecksum == "" {
-			continue
-		}
-		checksum, err := GetFlistChecksum(vm.Flist)
-		if err != nil {
-			return errors.Wrapf(err, "could not get flist %s hash", vm.Flist)
-		}
-		if vm.FlistChecksum != checksum {
-			return errors.Errorf("passed checksum %s of %s does not match %s returned from %s",
-				vm.FlistChecksum,
-				vm.Name,
-				checksum,
-				FlistChecksumURL(vm.Flist),
-			)
-		}
-	}
-	return nil
-}
-
-func (k *K8sCluster) ValidateMyceliumSeed() error {
```

Comment:
why removing mycelium seed validation?

Reply:
Because we are doing it in vm's validations and k8s node is just a vm